overlays/fixed-2clusters/emailservice.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: emailservice
```

Comment:
Would add an appropriate label too, to simplify selecting local or remote pods if necessary.